_data/hosts.yml

```diff
@@ -143,6 +143,17 @@
   php54: 35
   default: 5.3.29 
 
+- name: "Uberspace.de"
+  url: "https://uberspace.de/"
+  php52: 17
+  php53: 29
+  php54: 35
+  php55: 19
+  php56: 3
+  default: ??
+  manual_update: No
+  auto_update: Yes
```

Comment:
For now, this seems to mean that auto-update works for patch version (when choosing ``5.3``)